third_party/silabs/efr32_sdk.gni

```diff
@@ -204,11 +205,21 @@ template("efr32_sdk") {
       "${sl_ot_efr32_root}",
     ]
 
+    if (silabs_family == "efr32mg24") {
+    _include_dirs += [
+      "${efr32_sdk_root}/platform/Device/SiliconLabs/EFR32MG24/Include",
+      ]
+    }
+
     if (use_SiWx917) {
       _include_dirs += [
-        "${wifi_sdk_root}/components/si91x/inc",
         "${wifi_sdk_root}/components/si91x/memory",
-        "${wifi_sdk_root}/components/si91x/sl_net/inc",
+        "${wifi_sdk_root}/components/device/silabs/si91x/wireless/inc",
```

Comment:
these should be added in rs9117.gni file right
There is no point of adding this in multiple files

Reply:
Added files in efr32_sdk.gni only if you add it int rs9117.gni it's giving build error for some include files